Alerts improvements

viveleroi opened the pull request:
- Adds item use alerts. Closes #42
- Adds alert cap. Closes #41
- Minor config/locale standardizations

viveleroi commented:
@LXGaming Any thoughts?

Seems fine to me, although would we better be served by a name like `PrismRecordPreSave` or something indicating this occurs prior to a save?

pietelite commented:
I agree! Let me know if there's anything else.

LXGaming commented:
Yeah this seems alright, However I am curious as to how many events this would create and if that would cause issues with Sponge but in theory it should be fine.

pietelite commented:
> Yeah this seems alright, However I am curious as to how many events this would create and if that would cause issues with Sponge but in theory it should be fine.

I don't think that should be an issue considering how many events Sponge throws around in the event manager.

viveleroi committed:
Alerts improvements

- Adds item use alerts. Closes #42
- Adds alert cap. Closes #41
- Minor config/locale standardizations
